src/containers/reposts-page/store/sagas.ts

```diff
@@ -36,13 +43,25 @@ const getTrackReposts = createUserListProvider<Track>({
   fetchAllUsersForEntity: ({
     limit,
     offset,
-    entityId
+    entityId,
+    currentUserId
   }: {
     limit: number
     offset: number
     entityId: ID
-  }) =>
-    AudiusBackend.getRepostersForTrack({ limit, offset, trackId: entityId }),
+    currentUserId: ID | null
+  }) => {
+    const trackId = encodeHashId(entityId)!
```

Comment:
let's avoid using force unwraps - we should throw an error if the ID is unencodable for some reason

Reply:
same comment for other one